osu.Game.Rulesets.Osu/Mods/OsuModApproachCircle.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Bindables;
+using osu.Framework.Extensions.IEnumerableExtensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Osu.Objects.Drawables;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    internal class OsuModApproachCircle : Mod, IApplicableToDrawableHitObjects
+    {
+        public override string Name => "Approach Circle";
+        public override string Acronym => "AC";
+        public override string Description => "Never trust the approach circles...";
+        public override double ScoreMultiplier => 1;
+        public override IconUsage? Icon { get; } = FontAwesome.Regular.Circle;
+
+        [SettingSource("Easing", "Change the easing type of the approach circles.", 0)]
+        public Bindable<Easing> BindableEasing { get; } = new Bindable<Easing>();
```

Comment:
This can't be a setting in its current state. The enum member names are unreadable to non-programmers. It's bad UX to display to the user things like `OutQuint`, `InSine`, etc. - use human-readable names.